toDoApp/app/Http/Controllers/CardsController.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Model\Card;
+use App\Http\Requests\CreateCardRequest;
+use App\Http\Requests\UpdateCardRequest;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Http\Response;
+
+
+class CardsController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $cards = Card::where('user_id', auth()->user()->id)->get();
```

Comment:
napravio si relaciju `cards()` na useru, ovde treba to da koristis
znaci:
`auth()->user()->cards`
`auth()->user()->cards()` ti je query builder ukoliko hoces jos po necemu da filtriras, sortiras ili sta god na sql nivou a 
`auth()->user()->cards` ti izvrsi taj query, kao da si rekao `auth()->user()->cards()->get()`